ec2/sc-ec2-linux-jumpcloud-notebook-v1.0.0.yaml

```diff
@@ -438,3 +438,5 @@ Outputs:
     Value: "https://sagebionetworks.jira.com/wiki/spaces/IT/pages/996376579/Connect+to+Provisioned+Instances"
   ConnectionURI:
     Value: !Sub "https://${AWS::Region}.console.aws.amazon.com/systems-manager/session-manager/${LinuxInstance}?region=${AWS::Region}"
+  RStudio:
```

Comment:
I think this should be something like 'NotebookConnectionURI', but up to you

Reply:
Agree with Aaron --  we will eventually give them a choice of this being Rstudio or Jupyter.

Reply:
done